<?xml version="1.0"?>

<robot name="roundbot_gps_sim" xmlns:xacro="http://www.ros.org/wiki/xacro">

  <xacro:include filename="$(find roundbot_description)/urdf/roundbot.urdf.xacro" />
  <xacro:include filename="$(find ugv_lecture_gazebo)/urdf/gps.urdf.xacro" />

  <xacro:gps link_name="gps" parent_link="base_footprint" x="0.0" y="0.0" z="0.65" ref_lat="$(arg ref_lat)" ref_lon="$(arg ref_lon)" />
</robot>
